deployments/goerli/usdc/deploy.ts

```diff
@@ -24,32 +24,6 @@ async function deployContracts(deploymentManager: DeploymentManager, deploySpec:
   // Deploy governance contracts
   const { COMP, fauceteer, timelock } = await cloneGov(deploymentManager);
 
-  const usdcProxyAdmin = await deploymentManager.deploy('USDC:admin', 'vendor/proxy/transparent/ProxyAdmin.sol', []);
-  const usdcImpl = await deploymentManager.clone('USDC:implementation', clone.usdcImpl, []);
```

Comment:
You can also remove `usdcImpl` and `usdcProxy` from `clone`.

Reply:
he's referring to lines 6 and 7:

```
const clone = {
  usdcImpl: '0xa2327a938Febf5FEC13baCFb16Ae10EcBc4cbDCF',
  usdcProxy: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
```

Reply:
Fixed